contrib/checkbox-ce-oem/checkbox-provider-ce-oem/data/video-codec-test-confs/genio-510.json

```
@@ -0,0 +1,15 @@
+{
+    "gst_v4l2_audio_video_synchronization": {
+        "video_sinks": {
+            "desktop": "waylandsink",
+            "non_desktop": "kmssink connector-id=32 driver-name=mediatek"
```

can we separate those sinks attributes?
```suggestion
            "on_desktop": "waylandsink",
            "on_server": "kmssink connector-id=32 driver-name=mediatek",
            "on_core": "kmssink connector-id=32 driver-name=mediatek",
```

---

Yes, it's good suggestion.